source/index.ts

```diff
@@ -63,17 +62,23 @@ if (
 }
 
 class Chromium {
+  /**
+   * Determines the headless mode that chromium will run at
+   * https://developer.chrome.com/articles/new-headless/#try-out-the-new-headless
+   * @values true or "new"
+   */
+  static headlessMode: true | "new" = "new";
```

Comment:
First off, I think non-headless has been broken for a while. I believe once I switch to the `headless.gn` build variables, non-headless stopped working. I am working on some docs to show users how to access a headful version of chromium.

To address your comment:
I was having problems with a getter and setter, so I made the variable public, but kept the getter to preserve backwards compatibility. The variable also needs to be public so it can be used in puppeteer.launch, it needs to be accessed outside the class and not only internally.

I just rewrote some things. I think I was having problems because I want to keep the library with all the variables and functions static, so users don't need to instantiate the class before using it. Looks like all is working now though.

Resolved in 0c1351f01b021d961e5bbf497350247f679db2df and 3f6eb9ee91b1fb84d163432dd691b66b8758c980